misc/docker-ci/check.mk

```diff
@@ -1,9 +1,9 @@
 CONTAINER_NAME=h2oserver/h2o-ci:ubuntu1604
 SRC_DIR=/h2o
 CHECK_MK=$(SRC_DIR)/misc/docker-ci/check.mk
-CMAKE_ARGS=
-BUILD_ARGS=
-TEST_ENV=
+CMAKE_ARGS?=
```

Comment:
Do we want to add the flags here instead?

I say this because what we can test depends on the images that we control.

Reply:
Ah, yes, we don't have to make things too complicated. Fixed: 01cb9dc8cce712631f5fd5df706aa9391fe0c0fd